<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsChatClass</class>
 <widget class="QMainWindow" name="UsChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>492</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>492</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>492</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>UsChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UsChat.qrc">
    <normaloff>:/Icon/Resources/Icon/WxIcon.ico</normaloff>:/Icon/Resources/Icon/WxIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="PaintWidget">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>492</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>492</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="bar_widget" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-1</y>
      <width>350</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>42</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>42</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius:1px;
</string>
    </property>
    <widget class="QLabel" name="lb_headName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 11pt &quot;Akzidenz-Grotesk BQ Light with &quot;;
color: rgb(160, 160, 160);</string>
     </property>
     <property name="text">
      <string>UsChat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_close">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>0</y>
       <width>43</width>
       <height>33</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>43</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>43</width>
       <height>33</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color:rgb(224,84,84);
}
QPushButton:pressed{
	background-color:rgb(220,72,72);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="UsChat.qrc">
       <normaloff>:/Icon/Resources/Icon/main_CLOSE.png</normaloff>:/Icon/Resources/Icon/main_CLOSE.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_setUser">
     <property name="geometry">
      <rect>
       <x>263</x>
       <y>1</y>
       <width>43</width>
       <height>33</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>43</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>43</width>
       <height>33</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color:rgb(227,227,227);
}
QPushButton:pressed{
	background-color:rgb(205,205,205);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="UsChat.qrc">
       <normaloff>:/Icon/Resources/Icon/main_Settings.png</normaloff>:/Icon/Resources/Icon/main_Settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>13</width>
       <height>13</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="login_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>350</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <widget class="QLabel" name="lb_pic">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>80</y>
       <width>113</width>
       <height>113</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>113</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>113</width>
       <height>113</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,50);
border-radius:3px;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="UsChat.qrc">:/Icon/Resources/Icon/WxIcon.ico</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_userName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 12pt &quot;YouYuan&quot;;
color:rgb(120, 120, 120);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_login">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>280</y>
       <width>263</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>263</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>263</width>
       <height>50</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(26, 173, 25);
	color: rgb(245, 245, 245);
	font: 11.5pt &quot;YouYuan&quot;;
	border-radius:1px;
	font-weight:bold;
}
QPushButton:hover{
    background-color:rgb(18,150,17);
	
}
QPushButton:pressed{
	background-color: rgb(26, 173, 25);
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_changeUser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>380</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 11pt &quot;幼圆&quot;;
color: rgb(100, 104, 166);
font-weight:normal;</string>
     </property>
     <property name="text">
      <string>切换账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>350</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <widget class="QPushButton" name="bt_clearLoginInfo">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>90</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 10pt &quot;幼圆&quot;;
	background-color: rgb(244, 84, 84);
	color:rgb(255,255,255);
	border-radius:1px;
}
QPushButton:hover{
	background-color:rgb(224,84,84);
}
QPushButton:pressed{
	background-color:rgb(220,72,72);
}</string>
     </property>
     <property name="text">
      <string>清除登录信息</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_settingheader">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;幼圆&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="newUser_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>350</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>450</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 220);</string>
    </property>
    <widget class="QLabel" name="lb_newuserheader">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>9</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 19pt &quot;幼圆&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(107, 107, 107);</string>
     </property>
     <property name="text">
      <string>请登录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_user">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>92</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{ 
	background:rgba(255,255,255,0%); 
	border:0px;
	color: rgb(80, 80, 80);
	font: 11pt &quot;幼圆&quot;;
}
QLineEdit:hover{ 

}
QLineEdit:!hover{
	background:rgba(0,0,0,0%);
}
QLineEdit:focus{
border-radius:2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>171</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{ 
	background:rgba(255,255,255,0%); 
	border:0px;
	color: rgb(80, 80, 80);
	font: 11pt &quot;幼圆&quot;;
}
QLineEdit:hover{ 

}
QLineEdit:!hover{
	background:rgba(0,0,0,0%);
}
QLineEdit:focus{
border-radius:2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_newuserlogin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>295</y>
       <width>263</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>263</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>263</width>
       <height>50</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(26, 173, 25);
	color: rgb(245, 245, 245);
	font: 11.5pt &quot;YouYuan&quot;;
	border-radius:1px;
	font-weight:bold;
}
QPushButton:hover{
    background-color:rgb(18,150,17);
	
}
QPushButton:pressed{
	background-color: rgb(26, 173, 25);
}</string>
     </property>
     <property name="text">
      <string>请输入用户名和密码</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_saveInfo">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>236</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(122, 122, 122);
font: 9pt &quot;幼圆&quot;;</string>
     </property>
     <property name="text">
      <string>记住登录信息</string>
     </property>
    </widget>
    <widget class="Line" name="line_up">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>124</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(140, 140, 140);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_down">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>203</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(140, 140, 140);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_userText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>137</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(148, 148, 148);
font: 8pt &quot;幼圆&quot;;</string>
     </property>
     <property name="text">
      <string>请输入用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_passText">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>216</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(148, 148, 148);
font: 8pt &quot;幼圆&quot;;</string>
     </property>
     <property name="text">
      <string>请输入密码</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UsChat.qrc"/>
 </resources>
 <connections/>
</ui>
